src/serializer/Emitter.js

```diff
@@ -268,9 +269,22 @@ export class Emitter {
     invariant(this._activeValues.has(value));
     return condition ? value : undefined;
   }
-  emitAfterWaiting(delayReason: void | Value | SerializedBody, dependencies: Array<Value>, func: () => void) {
+  emitAfterWaiting(
+    delayReason: void | Value | SerializedBody,
+    dependencies: Array<Value>,
+    func: () => void,
+    targetBody: ?SerializedBody
```

Comment:
`?SerializedBody` allows for either `undefined` or `null`. We don't generally do that in Prepack. (We also don't use `== null` comparisons.) I'd prefer `void | SerializedBody`, or `targetBody?: SerializedBody` if you want to allow omitting the last argument (that way it still rules out `null`).

Reply:
Will do.